docs/security.html

```diff
@@ -273,6 +275,24 @@
     sasl.kerberos.service.name="kafka"</pre>
             </li>
         </ol></li>
+
+    <li><h4><a id="security_rolling_upgrade" href="#security_rolling_upgrade">Incorporating Security Features in a Running Cluster</a></h4>
+        You can upgrade a running cluster to be secured by one of the supported protocols discussed previously. This is done in two phases:
+        <ul>
+            <li>Rolling upgrade the cluster to open an additional, secured port.</li>
+            <li>Rolling upgrade again to close the plaintext port.</li>
+        </ul>
+        The steps for configuring SSL or SASL are described in sections <a href="#security_ssl">7.2</a> and <a href="#security_sasl">7.3</a>. These should be followed to upgrade the cluster to be secured appropriately using an additional port. The plaintext port must be left open so brokers can communicate during the rolling bounce.
+        So for example when upgrading to SSL, in the first rolling upgrade, the additional secured port is opened by configuring brokers with an additional listener:
+        <pre>
+        listeners=PLAINTEXT://host.name:port,SSL://host.name:port
+        </pre>
+        In the second rolling upgrade the plaintext port is closed by removing it from the listeners config:
+        <pre>
+        listeners=SSL://host.name:port
+        </pre>
+        If Zookeeper security is required, upgrade zookeeper nodes after the main cluster has been upgraded. The steps for doing this are covered in section <a href="#zk_authz_migration">7.5.2</a>.
```

Comment:
What is the reasoning for securing the zookeeper nodes after the main cluster has been secured?